administration/src/bp-modules/applications/__tests__/ApplicationStatusBar.test.tsx

```diff
@@ -118,11 +119,11 @@ describe('ApplicationStatusBar', () => {
         activeBarItem={barItems[0]}
       />
     )
-
+    await act(async () => null) // Blueprint5.Icon update
```

Comment:
there is a general issue with the Blueprint icons that lead to a console.error
Not sure if we can find a general solution for that instead of adding this in each test

Reply:
Mocking Icon/Button worked for me to avoid this error:
``` ts
jest.mock('@blueprintjs/core', () => ({
  ...jest.requireActual('@blueprintjs/core'),
  Icon: () => 'icon',
  Button: () => 'button',
}))
```
You could replace these strings with actual mocks to still be able to assert/test on specific button behavior. You could then even put this to the `jest.setup.ts` file to avoid having to mock it separately in every file.

One note: For me actually only the first test in this file logged an error, everything else in this file had a clean output.

Reply:
i had to mock this in the particular test where its needed. 
If you put it in the `jest.setup` won't work since all buttons (where you have tests for) will only be a 'button' string. So no texts or anything can be checked

Reply:
> You could replace these strings with actual mocks

See my comment above. The code above was just an example how to handle this in general. To be able to normally assert you have to mock the button with some react component that actually calls the on click method.